src/reducers/accounts.js

```diff
@@ -71,7 +71,7 @@ export function getAccountById(
 
 export const globalBalanceHistorySelector = createSelector(
   getVisibleAccounts,
-  () => 20, // TODO this will come from settings probably
+  state => state.settings.chartTimeRange,
```

Comment:
I would create a selector on settings side and import it. exactly like for fiatUnitSelector. idea is that , from this reducers side, you don't really know where things are. this reducer is not aware of other stores structures